content/en/docs/concepts/policy/resource-quotas.md

```diff
@@ -28,7 +28,7 @@ Resource quotas work like this:
 - The administrator creates one `ResourceQuota` for each namespace.
```

Comment:
Optionally, remove the backticks here as well.

Reply:
Done.